releasenotes/notes/submit-logs-from-python-checks-4fa62927ae2b86a2.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    Implement API that allows Python checks to send logs for
```

Comment:
```suggestion
    Implements API that allows Python checks to send logs for
```

Reply:
I'm not sure this is desirable as these entries get directly put on the Agent release change log. In that context, it's less so that we are describing this specific patch but rather as a whole. Does that make sense?